tests/test_clients_integration.py

```diff
@@ -0,0 +1,190 @@
+import os
+
+import pytest
+from dotenv import load_dotenv
+
+from mode_client import ModeClient
+
+
+@pytest.fixture
+def report_id():
+    return os.getenv("MODE_REPORT_ID")
+
+
+@pytest.fixture
+def space_id():
+    return os.getenv("MODE_SPACE_ID")
+
+
+@pytest.fixture
+def query_id():
+    return os.getenv("MODE_QUERY_ID")
```

Comment:
For more complicated methods or methods that can modify the entry, it is required to explicitly pass a specific identifier via an ENV variable.
These tests are not run in CI or anywhere else, so they are only run manually.

Reply:
> These tests are not run in CI or anywhere else, so they are only run manually.

yeah, seems like we don't really have a CI setup for this repo?
wondering do you think setting up a simple CI for running these test for this repo would be a good idea? since now we have these tests

Reply:
yes, we dont have a working CI setup for this repo, I can work on working one for lint and unit tests

Reply:
I will create a ticket

Reply:
> I can work on working one for lint and unit tests

sounds good! yeah just these very simple checks for now would be good enough